Assets/Scripts/Luke/InteractionBehaviour.cs

```diff
@@ -8,8 +8,16 @@ namespace Luke
 {
     public class InteractionBehaviour : MonoBehaviour, IInteractable
     {
-        public IInteractor Interactor { get => throw new System.NotImplementedException(); set => throw new System.NotImplementedException(); }
-        public UnityEvent Response { get => throw new System.NotImplementedException(); set => throw new System.NotImplementedException(); }
+        public IInteractor Interactor
+        {
+            get { return Interactor;}
```

Comment:
this will cause an infinite loop. The Behaviour needs to declare a local member that it returns.